tests/search/empty_fields_in_response/empty_fields_in_response.rb

```diff
@@ -9,16 +9,31 @@ def setup
     set_description("Test empty single- and multivalued fields in search and get")
   end
 
+  def self.final_test_methods
+    [ "test_empty_fields_in_search_and_get_response_elasticonly" ]
+  end
+
   def test_empty_fields_in_search_and_get_response
+    @doctype = 'test'
     deploy_app(SearchApp.new.sd(selfdir+"test.sd").enable_document_api)
     start
     feed_and_wait_for_docs("test", 4, :file => selfdir + "docs.json")
     assert_search
     assert_get
   end
 
+  def test_empty_fields_in_search_and_get_response_elasticonly
```

Comment:
Consider rename to _test_empty_reference_field_in_search_and_get_response_ (or similar).